### PR TITLE
Market maker / Exchange bug fixes

### PR DESCRIPTION
**Pull request by NotEternal**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/MultiCurrencyWallet/blob/master/docs/CONTRIBUTING.md) guide
- [ ] Good naming (as clear and simple as possible)
- [ ] Correct behavior if external API endpoints are down, return 404, 504 (or no answer), 401 errors (ddos simulation)
- [ ] I tested desktop/mobile resolution
- [x] I tested light/dark theme
- [ ] I tested different languages
- [ ] I checked the functionality once again (**AFFECT MONEY**)
- [x] I checked work on the Testnet
- [x] I checked work on the Mainnet
- [ ] I checked the **PR** once again

### CODE REVIEW

File: src/front/shared/pages/CreateWallet/Steps/StepsWrapper.tsx [hunk 1]
```diff
@@ -23,12 +23,9 @@ export default class StepsWrapper extends Component<any, any> {
     super(props)
     const { currencies } = props
     
-    if (config
-      && config.opts?.ownTokens
-      && Object.keys(config.opts.ownTokens)
-      && Object.keys(config.opts.ownTokens).length
-    ) {
+    if (config?.opts.ownTokens?.length) {
```
Comment (VitaliyShulik) [inline src/front/shared/pages/CreateWallet/Steps/StepsWrapper.tsx:26]: add `opts?`

File: src/front/shared/pages/Marketmaker/MarketmakerPromo.tsx [hunk 2]
```diff
@@ -107,17 +132,21 @@ export default class MarketmakerPromo extends React.Component<{}, {}> {
...
-                (BNB)WBTC
-              </Link>
-              <Link to={'/marketmaker/{matic}wbtc'} styleName="pseudLink">
-                (MATIC)WBTC
-              </Link>
-            </div>
+            {userTokens.length ? (
+              <div styleName="pseudLinksWrapper">
+                {userTokens.map((token, index) => {
+                  if (token.includes('wbtc')) {
```
Comment (VitaliyShulik) [inline src/front/shared/pages/Marketmaker/MarketmakerPromo.tsx:138]: BTCB исключаем?

Comment (NotEternal) [reply src/front/shared/pages/Marketmaker/MarketmakerPromo.tsx]: спасибо, забыл про него

File: src/front/shared/pages/CreateWallet/Steps/StepsWrapper.tsx (hunk 1 above)
Comment (NotEternal) [reply src/front/shared/pages/CreateWallet/Steps/StepsWrapper.tsx:26]: это точно будет обьект

**Review by VitaliyShulik — APPROVED** [review]

**Pull request merged**